protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/record/value/usertask/UserTaskRecord.java

```diff
@@ -70,14 +72,15 @@ public final class UserTaskRecord extends UnifiedRecordValue implements UserTask
       new ArrayProperty<>("changedAttributes", StringValue::new);
 
   public UserTaskRecord() {
-    super(16);
+    super(17);
```

Comment:
:wrench: We'll need to adjust this number depending if https://github.com/camunda/zeebe/pull/15915#pullrequestreview-1862549968 is already merged.

Reply:
Yep, I hate PRs that depend on each other 😆

Reply:
Done after rebasing onto `main`